modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/filters/implementation/DateRangeFilter.tsx

```diff
@@ -65,7 +66,7 @@ const getIsoString = ({
 
 	const dateISOString = date.toISOString();
 
-	if (entityFieldType === 'date') {
+	if (entityFieldType === EEntityFieldType.DATE) {
```

Comment:
On this line, there is a fork in odata formats. We make the `.split('T')[0]` for the `date` entity type, and leave ISO-format string as it is for the `date-time` format.